packages/jest-phabricator/src/index.js

```diff
@@ -53,9 +53,27 @@ module.exports = function(results: AggregatedResult): PhabricatorReport {
   const formatter = (coverage, reporter) => coverageMap;
   const report = formatTestResults(results, formatter);
 
-  return Object.assign((Object.create(null): any), results, {
+  const returnPhabricatorReport: PhabricatorReport = {
```

Comment:
Hey, welcome to Facebook :)

Actually, the difference would simple to change it to:

```
return {
  <all your object data>
};
```

that should work!

I recommend you to do the following:
* `git checkout .` (to get rid of those changes)
* `git pull --rebase upstream master` or whatever you map facebook/jest to, to get the latest changes.
* Run `yarn` on the top level of the repo. If that fails, run it again. There are some concurrency issues sometimes.

That should normally work and get you into a good state :)